spark/src/main/scala/geotrellis/spark/timeseries/RDDTimeSeriesMethods.scala

```diff
@@ -51,18 +51,49 @@ import java.time.ZonedDateTime
   @experimental def meanReduction(left: MeanResult, right: MeanResult): MeanResult =
     left + right
 
+  /**
+    * $experimental
+    */
+  @experimental def maxProjection(tile: Tile): Double = {
```

Comment:
These implementations are direct copies of the `handleFullTile` methods on their respective polygonal summary handler classes that were removed in favor of the new API.